studio/components/interfaces/GraphQL/GraphiQL.tsx

```diff
@@ -56,7 +56,10 @@ const GraphiQL = ({ fetcher, theme = 'dark', apiKey }: GraphiQLProps) => {
   }
 
   return (
-    <GraphiQLProvider fetcher={fetcher} defaultHeaders={JSON.stringify({ apiKey }, null, 2)}>
+    <GraphiQLProvider
+      fetcher={fetcher}
+      defaultHeaders={JSON.stringify({ Authorization: `Bearer ${accessToken}` })}
```

Comment:
We'll only want to show the headers if an access token exists to edit. Something like `defaultHeaders={accessToken ? JSON.stringify({ Authorization: `Bearer ${accessToken}` }) : undefined}`